packages/beacon-node/src/chain/chain.ts

```diff
@@ -795,6 +800,7 @@ export class BeaconChain implements IBeaconChain {
     this.seenAttestationDatas.onSlot(slot);
     this.reprocessController.onSlot(slot);
 
+    this.logger.debug("Pruning produced block caches for slot", {slot});
```

Comment:
Is this log redundant to "clock slot"?

Reply:
yes, i guess the inference can be made from clock slot, will remove post debugging